hadoop-tools/hadoop-aws/src/main/java/org/apache/hadoop/fs/s3a/InconsistentS3ClientFactory.java

```diff
@@ -18,48 +18,98 @@
 
 package org.apache.hadoop.fs.s3a;
 
-import com.amazonaws.ClientConfiguration;
```

Comment:
have we set `fs.s3a.s3.client.factory.impl` to `org.apache.hadoop.fs.s3a.InconsistentS3ClientFactory` and run the test suite and checked that this works (is throwing 503's)?